examples/rust/get_started/examples/05-secure-channel.rs

```diff
@@ -1,29 +1,31 @@
 // This node creates a secure channel and routes a message through it.
 
-use ockam::{Context, Result, Route, SecureChannel, Vault};
+use ockam::{Address, Context, LocalEntity, Result, Route, SecureChannel, Vault};
 use ockam_get_started::Echoer;
 
 #[ockam::node]
 async fn main(mut ctx: Context) -> Result<()> {
     // Start an Echoer worker at address "echoer"
-    ctx.start_worker("echoer", Echoer).await?;
-
-    let vault = Vault::create(&ctx)?;
+    let mut local = LocalEntity::create_with_worker(&ctx, "echoer", Echoer).await?;
```

Comment:
Let's have consistent names, like initiator&responder/alice&bob/etc.

Reply:
This was the original word from the guide. I agree we should be consistent but didn't want to change the original too much.